src/studio-header/NavDropdownMenu.jsx

```diff
@@ -30,7 +30,7 @@ const NavDropdownMenu = ({
 
 NavDropdownMenu.propTypes = {
   id: PropTypes.string.isRequired,
-  buttonTitle: PropTypes.string.isRequired,
+  buttonTitle: PropTypes.node.isRequired,
```

Comment:
Not related to this feature. This is a fly-by fix of a console warning.

Reply:
[clarification/suggestion] If we make the change here, should we also ensure all other prop type definitions for `buttonTitle` reflect `PropTypes.node`, too?

For example, [here](https://github.com/openedx/frontend-component-header/blob/e44001e94553d377a60f341028394679b2ab06df/src/studio-header/StudioHeader.jsx#L59) and [here](https://github.com/openedx/frontend-component-header/blob/e44001e94553d377a60f341028394679b2ab06df/src/studio-header/HeaderBody.jsx#L150).

Reply:
Fixed other places here: 06bfcc706e719b347c6df7daebc333592c9b68a2